core/kazoo_number_manager/src/knm_tasks_listener.erl

```diff
@@ -0,0 +1,176 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013-2016, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Pierre Fenoll
+%%%-------------------------------------------------------------------
+-module(knm_tasks_listener).
+-behaviour(gen_listener).
+
+-export([start_link/0]).
+
+-export([help/2]).
+
+-export([init/1
+         ,handle_call/3
+         ,handle_cast/2
+         ,handle_info/2
+         ,handle_event/2
+         ,terminate/2
+         ,code_change/3
+        ]).
+
+-include("knm.hrl").
+
+-define(SERVER, ?MODULE).
+
+-record(state, {}).
+
+-define(BINDINGS, [{'tasks', [{'restrict_to', ['help']}
+                             ]
+                   }
+                  ]).
+-define(RESPONDERS, [{{?MODULE, 'help'}, [{<<"tasks">>, <<"help_req">>}]}
+                    ]).
+-define(CONSUME_OPTIONS, [{'exclusive', 'false'}]).
```

Comment:
they would with bindings and the benefit would be to have a known number off apps replying